slayer/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -578,9 +578,17 @@ else if (npcDialog == null)
 			}
 			forcedWait--;
 		}
+		
 
-		if (infoTimer != null && config.statTimeout() != 0)
+		// If a timeout is configured for showing slayer stats
+		if (config.statTimeout() != 0)
 		{
+			// If the timer has not started, start it now
+			if (infoTimer == null)
+			{
+				infoTimer = Instant.now();
+			}
```

Comment:
could you not just do:

`if (config.statTimeout() != 0 &&infoTimer == null )
	{
		infoTimer = Instant.now();
	}`?

Reply:
That is an option, and that is how i structured it originally. I would argue that the my current way is neater than having it like this:

```
if (config.statTimeout() != 0 && infoTimer == null ) { infoTimer = Instant.now(); }

if (config.statTimeout() != 0 )
{
    // ...
}
```

Reply:
Yeah your right. It would also help if I looked at the whole of to see there was more code there :)